src/views/AuthorityView/AuthorityView.js

```diff
@@ -46,23 +39,28 @@ const propTypes = {
   }).isRequired,
 };
 
-const AuthorityView = ({
-  marcSource,
-  authority,
-}) => {
+const AuthorityView = ({ marcSource, authority }) => {
+  const [requestDeletion, setRequestDeletion] = useState(false);
```

Comment:
I think it's better to call it `deleteModalOpen` so it's more clear what this variable is